### PR TITLE
Add underscore to js evaluation to allow for use of get()

### PR DESCRIPTION
**Pull request by macebake**
This will help with nested fields that could potentially be `null`

### CODE REVIEW

**Comment by LHeggy** [comment]
##### On Performance

after pulling up vm creation to an init function it is way way faster in every sense:
```
1.03s user 1.03s system 188% cpu 1.094 total
```
(tested on 1.5k entries)

##### On shared state

I reckon overwriting the subject each operation will protect us from any serious issues and further effort is not warranted because:
- Checked all 1.5k entries made and they match with entries in the database, so nothing is getting garbled by reusing the VM (at least in this simple use case)
- [Someone went down the rabbit hole of making this safe a few years ago](https://github.com/Comcast/rulio/issues/77) and it looks like they gave up. Although it seems like it was an academic concern even in their use case
- I can't imagine what a user would have to be doing with their jsonnet file to really start leaving nasty state in our otto VM. If that happens in the next few months I feel like we can tell them not to get clever with their jsonnet files and be a normal egg.

**Comment by LHeggy** [comment]
> Something isn't quite right here - when I test locally on the backstage samples, only catalog types defined without a filter are synced. Filters are checked in `output/collect.go`

tweaked the jsonnet file, should be working now

File: docs/backstage/importer.jsonnet
Comment (macebake) [inline docs/backstage/importer.jsonnet:1]: let's not commit this, it's still just meant for the example

File: docs/backstage/pipelines/entries.jsonnet [hunk 1]
```diff
@@ -10,7 +10,7 @@
     {
       'local': {
         files: [
-          'catalog-info.yaml',
+          'docs/backstage/catalog-info.yaml',
```
Comment (macebake) [inline docs/backstage/pipelines/entries.jsonnet:13]: same here

File: docs/backstage/pipelines/entries.jsonnet [hunk 2]
```diff
@@ -52,18 +52,18 @@
       type_name: 'Custom["BackstageAPI"]',
       source: {
         filter: '$.apiVersion == "backstage.io/v1alpha1" && $.kind == "API"',
-        name: '$.metadata.name',
-        external_id: '$.metadata.namespace + "/" + $.metadata.name',
+        name: '_.get($.metadata, "name", "default API name")',
+        external_id: '_.get($.metadata, "namespace", "default description") + "/" + _.get($.metadata, "name", "default API name")',
         aliases: [
-          '$.metadata.name',
+          '_.get($.badKey, "name", "default alias")',
```
Comment (macebake) [inline docs/backstage/pipelines/entries.jsonnet:58]: did you mean to commit `$badKey`? 

also, in these cases, we're sure that accessing nested field as normal still works? i assume so but wanted to check. We probably don't need all the examples of `_.get()` in the sample docs IMO (lmk what you think). Hopefully it's clear from `expressions.md` that you can use both methods

File: docs/backstage/importer.jsonnet
Comment (LHeggy) [reply docs/backstage/importer.jsonnet:1]: removed

File: docs/backstage/pipelines/entries.jsonnet (hunk 2 above)
Comment (LHeggy) [reply docs/backstage/pipelines/entries.jsonnet]: I think that was just testing out aliases. I've replaced it with something less confusing

File: docs/backstage/pipelines/entries.jsonnet (hunk 1 above)
Comment (LHeggy) [reply docs/backstage/pipelines/entries.jsonnet]: removed

**Comment by macebake** [comment]
This looks good and all works for me! You can merge and cut a minor release. Also it's probably worth following up w Hank from Netflix (you can find this on the original thread)

**Review by ElizabethMRichardson — APPROVED** [review]

**Pull request merged**